addons/l10n_uk/demo/l10n_uk_demo.xml

```diff
@@ -1,26 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 <data noupdate="1">
-        <record id="smartmode" model="res.partner">
-            <field name="name">SmartMode LTD</field>
+        <record id="l10n_uk_1" model="res.partner">
```

Comment:
I don't think changing the xmlid of demo data in stable is a good thing. Any custom module whose demo data reference on it will break. This can be done, but only in master